openwisp_controller/config/migrations/0042_multiple_wireguard_tunnels.py

```diff
@@ -18,7 +18,11 @@ def allow_multiple_wireguard_tunneling(apps, schema_editor):
         vpn_id = template.vpn.pk.hex
         changed = False
         for interface in interfaces:
-            if interface['private_key'] not in [
+            interface_type = interface.get('type', None)
+            private_key = interface.get('private_key', None)
+            if interface_type not in ['wireguard', 'vxlan']:
```

Comment:
why vxlan, vxlan doesn't have private key, we can just skip if interface type is not wireguard or private key is not present

Reply:
Ok :+1: